### PR TITLE
Support github: field in extra-deps

### PR DESCRIPTION
**Pull request by fumieval**

### CODE REVIEW

**Comment by hasufell** [comment]
Can we add a test?

**Comment by hasufell** [comment]
ping

File: lib/StackageToHackage/Stackage/YAML.hs [hunk 1]
```diff
@@ -73,9 +73,10 @@ instance FromYAML GhcOptions where
 
 instance FromYAML Git where
   parseYAML = withMap "Git" $ \m -> Git
-      <$> m .: "git"
+      <$> (m .: "git" <|> github m)
       <*> m .: "commit"
       <*> m .:? "subdirs" .!= []
+    where github m = ("git@github.com:" <>) <$> (m .: "github")
```
[the remark this answers is not in the document]
Comment (fumieval) [reply lib/StackageToHackage/Stackage/YAML.hs]: Fixed. AFAIU it works without ".git"

**Comment by hasufell** [comment]
Released https://hackage.haskell.org/package/stack2cabal-1.0.13

**Comment by fumieval** [comment]
Thanks!

Btw I didn't touch the snapshot location stuff because specifying a git location isn't supported in the original code

**Pull request merged**